orchestra/workflow/load.py

```diff
@@ -0,0 +1,161 @@
+from django.db import transaction
+
+from orchestra.models import Certification
+from orchestra.models import Workflow
+from orchestra.models import WorkflowStep
+from orchestra.models import WorkflowVersion
+from orchestra.workflow.directory import parse_workflow_directory
+from orchestra.workflow.exceptions import WorkflowError
+
+
+@transaction.atomic
+def load_workflow(workflow_directory, versions=None, force=False):
+    data = parse_workflow_directory(workflow_directory)
+
+    # Create the workflow object if it doesn't exist
+    workflow_data = data['workflow']
+    workflow, workflow_created = create_or_update(
+        Workflow,
+        {'slug': workflow_data['slug']},
+        {
+            'name': workflow_data['name'],
+            'description': workflow_data['description'],
+            'code_directory': workflow_directory
+        }
+    )
+
+    # Create all certifications for the workflow
+    for certification_data in workflow_data['certifications']:
+        create_or_update(
+            Certification,
+            {
+                'slug': certification_data['slug'],
+                'workflow': workflow
+            },
+            {
+                'name': certification_data['name'],
+                'description': certification_data['description'],
+            }
```

Comment:
This (and the block above) avoid errors with loading different versions at different times (e.g., loading v1 creates the workflow, loading v2 shouldn't complain because the workflow exists). It also updates the workflow and its certifications based on the latest contents of the manifest.